### PR TITLE
generate error when no configuration is loaded

### PR DESCRIPTION
**Pull request by yukinying**
Validator should be fail-safe that if there is a misconfiguration that results in zero configurations loaded, it should generate an error.

This should not be confused with the use case where *one* of the directory is empty but some configurations are loaded. This pull request allow that use case.

### CODE REVIEW

**Comment by yukinying** [comment]
// cc @briantkennedy

File: pkg/gcv/configs/config.go [hunk 1]
```diff
@@ -491,6 +491,10 @@ func NewConfiguration(dirs []string, libDir string) (*Configuration, error) {
 		return nil, err
 	}
 
+	if len(unstructuredObjects) == 0 && len(regoLib) == 0 {
```
Comment (briantkennedy) [inline pkg/gcv/configs/config.go]: The regoLib files don't do anything on their own, so we should only be checking for the unstructuredObjects.

Comment (yukinying) [reply pkg/gcv/configs/config.go]: Fixed. I moved the logic into `loadUnstructured()` and removed the check on `regoLib`.

**Comment by yukinying** [comment]
Thanks. I have pull the latest changes from master. Could you merge the pull request?

**Pull request merged**